redisinsight/ui/src/pages/browser/components/stream-details/StreamDetailsWrapper.tsx

```diff
@@ -190,11 +190,12 @@ const StreamDetailsWrapper = (props: Props) => {
         return (
           <div>
             <PopoverDelete
+              header={id}
               text={(
                 <>
-                  Entry {id} will be removed from
-                  <br />
-                  {key}
+                  will be removed from
```

Comment:
W?

Reply:
Requirement changed, i don't know why . Made it same as hash, in hash "w"